src/gulpfile.js

```diff
@@ -255,6 +255,31 @@ gulp.task('build-html', ['add-revision-numbers'], function () {
   .pipe(gulp.dest(paths.build));
 });
 
+gulp.task('build-mseditor', ['build-scripts-template-editor', 'add-revision-numbers'], function () {
+  var sources = gulp.src([
+    paths.build + '/scripts/relayEditor.*.js'
+  ]);
+  return gulp.src([
+    paths.app + '/template-editor/index.html'
+  ])
+  .pipe(gulpInject(sources, {
+    addRootSlash: false, // ensures proper relative paths removing the root slash
+    ignorePath: paths.build // ensures proper relative paths removing the absolute path
+  }))
+  .pipe(gulp.dest(paths.build + '/template-editor'));
+});
+
+gulp.task('build-scripts-template-editor', function () {
+  return gulp.src([
+    paths.app + '/template-editor/*.js'
+  ])
+  .pipe(concat('relayEditor.js'))
```

Comment:
What is this file? why is it not in _template-editor_ folder?

Reply:
Ohh, is it the destination name?

Reply:
In that case could you use a lower case name?

Reply:
Yes it is the destination name, I will change for a lower case.